script/run-servers.sh

```diff
@@ -50,7 +51,7 @@ rm -rf $LOGS
 mkdir -p $LOGS
 
 echo "starting tri, port=${TRI_PORT}, log=${TRI_LOG}..."
-java -jar tri/TRIserver.jar [::1] $REGISTRAR_PORT \
+java -jar tri/TRIserver.jar [::1] $REGISTRAR_PORT $BORDER_AGENT_PORT -dyntx \
```

Comment:
Here it would be best to use the latest version of the TRI, which is v1.2. That version has the following help output if you run "java -jar TRIserver.jar" :  

```
TRIserver v1.2 - Thread spec v1.2.1-draft2 compliant
Usage:
     java -jar TRIserver.jar <RegistrarHostname> [<RegistrarPort>] [-log <logfile>]

Command line arguments:
  <RegistrarHostname>  Hostname of the Registrar to use. May be an IPv4 literal. May be an IPv6 literal in square brackets.
  <RegistrarPort>      Optional port number of the Registrar to use. If not given, the default 5684 is used.
  -log <logfile>       Optional filename/path of log file. If not given, defaults to 'tri.log'.

Example: to start a TRI that uses a localhost Registrar on the default DTLS-port:
     java -jar TRIserver.jar [::1]

Example: to start a TRI with non-local registrar on DTLS port 5684 with log file specified:
     sudo java -jar TRIserver.jar myregistrar.example.com 5684 -log /var/log/tri.log
```

So it doesn't have, nor require the -dyntx option, because that approach is used by default to be spec-compliant.
Because this could be confusing to people it seems best to upgrade to v1.2 and to include the TRIserver.jar in the archive here, because it seems to be not included in the current repo.

Reply:
thanks Esko. Let's keep it as is in this PR and update it in another PR. Thoughts? (Please feel free to create PR for it)

Reply:
Yes, that's ok - I will create a PR. Let's check with the TC before including the actual TRIserver. It could be placed e.g. in the 'script' folder.